src/NexusMods.Collections/Verbs.cs

```diff
@@ -42,4 +48,35 @@ private static async Task<int> InstallCollection([Injected] IRenderer renderer,
 
         return 0;
     }
+    
+    [Verb("gather-collection-definitions", "Downloads all the collection definitions for a given game, and extracts them to a folder")]
```

Comment:
Is this debug functionality? I don't see this used anywhere in the App or tests.
I think having this is fine, but it may be worth it to disable this from release builds; so the CLI is not cluttered for end users.

Reply:
Just for testing, I'll add some comments